server/routes/prisonerSelect/prisonerSelect.ts

```diff
@@ -30,6 +34,21 @@ export default class PrisonerSelectRoutes {
       user
     )
 
-    return this.renderView(req, res, { searchResults })
+    return this.renderView(req, res, { searchResults, searchTerm })
+  }
+
+  submit = async (req: Request, res: Response): Promise<void> => {
+    const { searchTerm } = req.body
+
+    const error = validateForm({ searchTerm })
+
+    if (error) return this.renderView(req, res, { error, searchTerm })
+
```

Comment:
Also, I don't see any tests for this replacing of a comma?

Reply:
Fair point, will add one to check the contents of the search form input.